politeiawww/api/www/v1/v1.go

```diff
@@ -1010,6 +1010,7 @@ type PolicyReply struct {
 	MaxLinkByPeriod            int64    `json:"maxlinkbyperiod"`
 	MinVoteDuration            uint32   `json:"minvoteduration"`
 	MaxVoteDuration            uint32   `json:"maxvoteduration"`
+	MinConfirmationsRequired   uint64   `json:"minconfirmationsrequired"`
```

Comment:
Can we change this to `PaywallConfirmations`?  I realize that `MinConfirmationsRequired` is what is called in the config, but that's not a very good name.

Reply:
Okay.